lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -58,8 +58,8 @@ class EntityRepository implements ObjectRepository, Selectable
     /**
      * Initializes a new <tt>EntityRepository</tt>.
      *
-     * @param EntityManager         $em    The EntityManager to use.
```

Comment:
Both of these docblock entries can be dropped, as they are redundant

Reply:
Yeah the descriptions aren't realy useful. I removed them.